tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "module": "esnext",
-    "moduleResolution": "node",
+    "module": "node16",
+    "moduleResolution": "node16",
```

Comment:
Why we need to change this? 🤔

Reply:
Else TS complains that it can't find the `execa` and `meow` modules. See https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

Reply:
Thanks for the reference!